src/stores/room-list-v3/skip-list/sorters/index.ts

```diff
@@ -24,7 +24,7 @@ export interface Sorter {
     /**
      * A string that uniquely identifies this given sorter.
      */
-    type: SortingAlgorithm;
+    type: string;
```

Comment:
As this is used by the RoomList and RoomSkipList to determine whether the sorter has changed – basically like a hash over the sorter's properties – this must uniquely identify all properties of the current sorter.